hyperbook/book/spiele/penguins/regeln.md

```diff
@@ -11,7 +11,14 @@ Hey, Danke für den Fisch ist ein Brettspiel, welches mit Figuren auf einem Hexa
 Am Anfang werden Fische zufällig auf alle hexagonalen Eisschollen verteilt, wobei es immer mindestens einen Fisch auf jeder Eisscholle gibt.
 
 :::alert{info}
-Auf dem Spielserver werden ``BOARD_SIZE * (BOARD_SIZE + 1)`` aka `8 * 9` viele Fische auf dem Board verteilt, wobei es auch zu Beginn Felder ohne Fische geben kann und nie mehr als 4 Fische auf einem Feld.
+Für die Erstellung des Spielfeldes gelten folgende Bedingungen:
+
+- Sei $x$ die Seitenlänge des zwei dimensionalen Spielfeldes, dann ist $x = 8$.
+- Das Spielfeld $S$ ist dann $S = x \times x = 8 \times 8 = 64$ Felder groß.
+- Auf dem Spielfeld können $\le140$ Fische platziert werden, wobei stets auf $\ge4$ Feldern keine Fische am Anfang sein werden.
+    - Es werden nur auf einer Hälfte des $64$ Felder großen Spielfeldes die Fische zufällig gelegt. 
+    - Um eine symethrische Spielerfahrung zu bieten, wird diese Hälfte dupliziertd und dann an seiner horizontalen und vertikalen Achse gespiegelt und den restlichen $32$ Feldern hinzugefügt.
+- Wenn $F_n$ ein Feld auf dem Spielbrett is, mit $0\le n\le 63$, dann sei $f_{F_n}$ die Zahl an Fischen auf einem Feld, mit $0\le f_{F_n}\le 4$.
```

Comment:
Ich finde das ein bisschen überkompliziert, wenn jemand es so genau wissen will soll er in den Code gucken, oder nicht?

Der interessante Part, dass außen mehr Fische und innen mehr Löcher sind, wurde aber ausgelassen?

Rechtschreibfehler: symethrische

Reply:
Es gibt auch nicht genau 4 Löcher, bitte schaut doch auch mal in der GUI drauf...

Reply:
> Ich finde das ein bisschen überkompliziert, wenn jemand es so genau wissen will soll er in den Code gucken, oder nicht?

Das Problem besteht nur, dass Leute die sich den Code ansahen trotz dessen diese Fragen hatten. 
Da nach einer relativ präzisen Dokumentation gefragt wurde, wollte ich dies damit lösen und als sehr kompliziert sehe ich das auch nicht.
Es ist leider wegen fehlender Dokumentation innerhalb des Codes auch nicht sehr ersichtlich, was tatsächlich da gemacht wird, weshalb @jnccd und ich versucht haben einen Reim darauf zu bilden. Dies hat dann leider zu einigen Falschaussagen unserseits geführt. 

> Der interessante Part, dass außen mehr Fische und innen mehr Löcher sind, wurde aber ausgelassen?

Wie bereits oben erwähnt, war dies wegen der fehlenden Dokumentation nicht ersichtlich.

> Rechtschreibfehler: symethrische

Wird korrigiert.


> Es gibt auch nicht genau 4 Löcher, bitte schaut doch auch mal in der GUI drauf...

Ich habe das Wort *mindestens* vergessen zu schreiben. Wenn ich @jnccd korrekt verstanden habe, dann sind es allerdings auch *mindestens 5 Felder*, korrekt? Ich bitte um Klärung.

Reply:
Nope, es kann auch gar keine Löcher geben, das ist bloß sehr unwahrscheinlich. Das ist bewusst etwas zufällig, damit man sich nicht einfache Strategien zurechtlegen kann ;)

> > Der interessante Part, dass außen mehr Fische und innen mehr Löcher sind, wurde aber ausgelassen?

> Wie bereits oben erwähnt, war dies wegen der fehlenden Dokumentation nicht ersichtlich.

Ich hatte euch gebeten beim backend ins changelog zu schauen dafür, da stand nämlich genau das drin was ich auch in der Dokumentation dann wollte :)

Heißt das ihr habt auch persönliche Anfragen als Tutoren bekommen, die diese Details erfragten?

Ja, der Code ist schlecht dokumentiert, das habe ich sehr eilig geschrieben - aber dann sagt Bescheid und ich hole das nach, dann müsst ihr euch nicht mühsam einen Reim drauf machen ;)

Reply:
> Ich hatte euch gebeten beim backend ins changelog zu schauen dafür, da stand nämlich genau das drin was ich auch in der Dokumentation dann wollte :)

Das war mich nicht bekannt, aber ich weiß das nun für die Zukunft.

>Ja, der Code ist schlecht dokumentiert, das habe ich sehr eilig geschrieben - aber dann sagt Bescheid und ich hole das nach, dann müsst ihr euch nicht mühsam einen Reim drauf machen ;)

Das wäre nicht nur für uns hervorragend, sondern auch für die Schüler.